aiofcm/connection.py

```diff
@@ -147,6 +162,26 @@ async def send_message(self, message):
         response = await future_response
         return response
 
+    async def send_ack(self, device_token, message_id):
+        if not self.connected:
+            await self.connect()
+        msg = aioxmpp.Message(
+            type_=aioxmpp.MessageType.NORMAL
+        )
+        payload = FCMMessage()
+
+        payload_body = {"to":str(device_token), "message_id":str(message_id), "message_type":"ack"}
```

Comment:
Let's make it a little bit more PEP8 friendly and add spaces between keys and values and split this dict to multiple lines because it's already has more than 80 characters.

Reply:
Fixed